contentgrid-spring-audit-logging/src/main/java/com/contentgrid/spring/audit/ContentGridAuditEventConfiguration.java

```diff
@@ -0,0 +1,71 @@
+package com.contentgrid.spring.audit;
+
+import com.contentgrid.spring.audit.extractor.AuditEventExtractor;
+import com.contentgrid.spring.audit.extractor.BasicAuditEventExtractor;
+import com.contentgrid.spring.audit.extractor.EntityContentEventExtractor;
+import com.contentgrid.spring.audit.extractor.EntityEventExtractor;
+import com.contentgrid.spring.audit.extractor.EntityItemCreateIdExtractor;
+import com.contentgrid.spring.audit.extractor.EntityItemEventExtractor;
+import com.contentgrid.spring.audit.extractor.EntityRelationEventExtractor;
+import com.contentgrid.spring.audit.extractor.EntitySearchEventExtractor;
+import com.contentgrid.spring.audit.handler.AuditEventHandler;
+import java.util.List;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
+import org.springframework.data.mapping.context.PersistentEntities;
+import org.springframework.data.rest.core.mapping.RepositoryResourceMappings;
+
+@Configuration(proxyBeanMethods = false)
+public class ContentGridAuditEventConfiguration {
+
+    @Bean
+    AuditObservationHandler auditObservabilityHandler(List<AuditEventExtractor> auditEventExtractors,
+            List<AuditEventHandler> auditEventHandlers) {
+        return new AuditObservationHandler(auditEventExtractors, auditEventHandlers);
+    }
+
+    @Bean
+    @Order(Ordered.LOWEST_PRECEDENCE)
```

Comment:
Currently my IDE wants to remove the value, because it is the default.

Have you considered letting `AuditEventExtractor` extend `Ordered` and giving it a different default ?
If you do, you can give a different default in the `BasicAuditEventExtractor` implementation